deployments/liqo/templates/webhooks/job-patch/job-create-secret.yaml

```
@@ -0,0 +1,38 @@
+{{- $cfg := (merge (dict "name" "webhook-certificate-patch" "module" "webhook-certificate-patch") .) -}}
+{{- $webhookConfig := (merge (dict "name" "webhook" "module" "webhook") .) -}}
+{{- $ctrlManagerConfig := (merge (dict "name" "controller-manager" "module" "controller-manager") .) -}}
+
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{ include "liqo.prefixedName" $cfg }}
+  annotations:
+    "helm.sh/hook": pre-install,pre-upgrade
+    "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
+  labels:
+    {{- include "liqo.labels" $cfg | nindent 4 }}
+spec:
+  ttlSecondsAfterFinished: 0
+  template:
+    metadata:
+      name:  {{ include "liqo.prefixedName" $cfg }}
+      labels:
+        {{- include "liqo.labels" $cfg | nindent 8 }}
+    spec:
+      containers:
+      - name: create
+        image: {{ .Values.webhook.patch.image }}
+        imagePullPolicy: {{ .Values.pullPolicy }}
+        args:
+        - create
+        - --host={{ include "liqo.prefixedName" $ctrlManagerConfig }},{{ include "liqo.prefixedName" $ctrlManagerConfig }}.{{ .Release.Namespace }}.svc
```

```suggestion
        - --host={{ include "liqo.prefixedName" $ctrlManagerConfig }},{{ include "liqo.prefixedName" $ctrlManagerConfig }}.{{ .Release.Namespace }},{{ include "liqo.prefixedName" $ctrlManagerConfig }}.{{ .Release.Namespace }}.svc,{{ include "liqo.prefixedName" $ctrlManagerConfig }}.{{ .Release.Namespace }}.svc.cluster.local
```

I'm not sure all Kubernetes distributions uses exactly the same hostname, it's better to include all the possibilities

---

I've honestly added the same elements as [ingress-nginx](https://github.com/kubernetes/ingress-nginx/blob/d29a2383507427abdc56302165c618bf2da7580b/charts/ingress-nginx/templates/admission-webhooks/job-patch/job-createSecret.yaml#L51) did, but adding more does not harm.